AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -2303,6 +2307,13 @@ public static void clearUserContent() {
     return featureFlagSet;
   }
 
+  @VisibleForTesting
+  public static Map<String, String> parseFileURLs(JSONObject response) {
+    JSONObject filesObject = response.optJSONObject(
```

Comment:
I think you need try catch here? Also null check, if the json key does not exists?

Reply:
good "catch" :), thanks!